### PR TITLE
sql/postgres: implement schema.Locker

### PR DESCRIPTION
**Pull request by a8m**
Please, don't review yet. Working on integration tests.

### CODE REVIEW

File: sql/postgres/driver_test.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+		require.Nil(t, unlock)
+	})
+}
+
+func TestDriver_UnlockError(t *testing.T) {
+	db, m, err := sqlmock.New()
+	require.NoError(t, err)
+	d := &Driver{}
+	d.ExecQuerier = db
+	name, hash := "up", 1551306158
```
Comment (yonidavidson) [inline sql/postgres/driver_test.go:74]: These hash numbers are just random, right? it's not calculated or something

**Review by yonidavidson — APPROVED** [review]
LGTM

File: sql/postgres/driver_test.go (hunk 1 above)
Comment (a8m) [reply sql/postgres/driver_test.go:74]: it's calculated (fnv), but it's here for testing.

**Review by hedwigz — APPROVED** [review]
LGTM

**Pull request merged**